generators/linting/templates/test/.eslintrc

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "springworks/mocha",
+  "globals": {
+    "should": false,
+    "sinon": false
+  },
+  "plugins": [
+    "should-promised"
+  ],
+  "rules": {
+    "should-promised/return-promise": 2
```

Comment:
I was unsure if it's really correct to generate with this rule. It's a good rule, but maybe it should be the default in our config instead?